www/source/javascripts/community.js

```diff
@@ -0,0 +1,25 @@
+$(function() {
+  if ($(".community-events")) {
+    var url = DEPOT_API + "/community-events";
+
+    $.ajax(url, {
+      success: function(data, status, xhr) {
+        if (typeof data === "object" && typeof data.length === "number") {
```

Comment:
You might consider  `if (Array.isArray(data))` here, if you're concerned about the data not being an array. (Although, should it ever not be?)